rafthttp/transport.go

```diff
@@ -111,6 +111,9 @@ type Transport struct {
 	// When an error is received from ErrorC, user should stop raft state
 	// machine and thus stop the Transport.
 	ErrorC chan error
+	// StreamDialRetryTimeout is used to alter the frequency of streamReader
+	// dial retrial attempts
+	StreamDialRetryTimeout time.Duration
```

Comment:
move this near to the dial timeout? also probably a retry rate limiter is better here?